bessctl/conf/samples/update_ttl.bess

```diff
@@ -0,0 +1,24 @@
+import scapy.all as scapy
+
+eth = scapy.Ether(src='02:1e:67:9f:4d:ae', dst='06:16:3e:1b:72:32')
+ip = scapy.IP(src='192.168.1.1', dst='10.0.0.1', ttl=2)
+udp = scapy.UDP(sport=10001, dport=10002)
+payload = 'helloworld'
+pkt_bytes = str(eth/ip/udp/payload)
+
+Source() -> Rewrite(templates=[pkt_bytes]) -> rr::RoundRobin(gates=[1, 2, 3])
+
+rr:1 \
+    -> UpdateTTL() \
```

Comment:
Put a little comment that says, UpdateTTL takes no parameters and just decrements that TTL by one, drops it if TTL == 0?